test/e2e/e2e.go

```diff
@@ -103,8 +103,9 @@ func setupProviderConfig() error {
 
 		cloudConfig.Provider = gceCloud
 
+		// Arbitrarily pick one of the zones we have nodes in
 		if cloudConfig.Zone == "" && framework.TestContext.CloudConfig.MultiZone {
-			zones, err := gceCloud.GetAllZones()
+			zones, err := gceCloud.GetAllCurrentZones()
```

Comment:
Is this going to work?  This is done early in e2e initialization, where client is not set yet?